rancher2/structure_cluster_v2.go

```diff
@@ -60,10 +60,18 @@ func flattenClusterV2(d *schema.ResourceData, in *ClusterV2) error {
 		d.Set("cloud_credential_secret_name", in.Spec.CloudCredentialSecretName)
 	}
 	if in.Spec.ClusterAgentDeploymentCustomization != nil {
-		d.Set("cluster_agent_deployment_customization", flattenAgentDeploymentCustomizationV2(in.Spec.ClusterAgentDeploymentCustomization))
+		flat, err := flattenAgentDeploymentCustomizationV2(in.Spec.ClusterAgentDeploymentCustomization)
```

Comment:
Nit: I know it seems verbose but be descriptive so I suggest

```
agentDeploymentCustomizationV2, err := flattenAgentDeploymentCustomizationV2(in.Spec.ClusterAgentDeploymentCustomization)
```

Reply:
Done.

Reply:
I don't see a new commit but I'll an eye out ;)

Reply:
I hadn't Pushed it yet. 
https://github.com/rancher/terraform-provider-rancher2/pull/1224/commits/fe71d96fffff23a6af9905367821df16de38f4d1